<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientExtractorDialog</class>
 <widget class="QDialog" name="ClientExtractorDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extracteur de client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainFrame.qrc">
    <normaloff>:/MainFrame/Resources/extract.png</normaloff>:/MainFrame/Resources/extract.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Dossier racine :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dir">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="replaceFiles">
     <property name="text">
      <string>Remplacer les fichiers existants</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteRes">
     <property name="text">
      <string>Supprimer les fichiers ressources (.res) apr√®s extraction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MainFrame.qrc"/>
 </resources>
 <connections/>
</ui>
